crates/dbsp/Cargo.toml

```diff
@@ -65,6 +65,7 @@ derive_more = { version = "0.99.17", features = ["add", "not", "from"] }
 feldera-storage = { path = "../feldera-storage" }
 tempfile = "3.8.1"
 ouroboros = "0.18.1"
+md5 = { version = "0.7.0" }
```

Comment:
why are we using md5? It's generally frowned upon, probably you're not using it in a security sensitive context but people see it and then freak out (even if it's not justified)